modules/wmco-upgrades.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * windows_containers/windows-node-upgrades.adoc
+
+[id="wmco-upgrades_{context}"]
+= Windows Machine Config Operator upgrades
+
+When a new version of the Windows Machine Config Operator (WMCO) is released that is compatible with the current cluster version, the Operator is upgraded based on the upgrade channel and subscription approval strategy it was installed with when using the Operator Lifecycle Manager (OLM). The WMCO upgrade results in the Kubernetes components in the Windows machine being upgraded.
+
+For a non-disruptive upgrade, the WMCO terminates the Windows machines configured by the previous version of the WMCO and recreates them using the current version. This is done by deleting the `Machine` object, which results in the drain and deletion of the Windows node. To facilitate an upgrade, the WMCO adds a version annotation to all the configured nodes. During an upgrade, a mismatch in version annotation results in the deletion and recreation of a Windows machine. To have minimal service disruptions during an upgrade, the WMCO ensures that the cluster has at least one Windows machine per MachineSet in the running state.
```

Comment:
>the WMCO ensures that the cluster has at least one Windows machine per MachineSet in the running state.

WMCO upgrades only one Windows Machine at a time.

@VaishnaviHire you should have updated https://github.com/openshift/windows-machine-config-operator#windows-nodes-kubernetes-component-upgrade when you made the changed the upgrade strategy.